README.md

```diff
@@ -33,18 +33,119 @@ We currently support [ONNX runtime](https://github.com/microsoft/onnxruntime) al
 | Pruning                            |        N/A            |    :heavy_check_mark:   |
 
 
-## Install
+## Installation
 
 🤗 Optimum can be installed using pip as follows:
 
-`pip install optimum`
+```bash
+python -m pip install optimum
+```
 
-🤗 Optimum with [Intel Neural Compressor (INC)](https://github.com/intel/neural-compressor) or [ONNX runtime](https://github.com/microsoft/onnxruntime) dependencies can be installed respectively using pip as follows:
+If you'd like to use the accelerator-specific features of 🤗 Optimum, you can install the required dependencies according to the table below:
 
-`pip install optimum[intel]`
+| Accelerator                                                                 | Installation                                 |
+|:----------------------------------------------------------------------------|:---------------------------------------------|
+| [ONNX runtime](https://github.com/microsoft/onnxruntime)                    | `python -m pip install optimum[onnxruntime]` |
```

Comment:
I decided to create a table here to accommodate all the upcoming integrations :)

Reply:
Great idea, I love it !

Reply:
Great idea, I love it !